teachers_digital_platform/jinja2/teachers_digital_platform/prototypes/crt-survey.html

```diff
@@ -1,148 +1,339 @@
-        <h3>Instructions</h3>
-        <p>Read through the scope and sequence of the curriculum, skim the lesson plans, and answer the following questions.</p>
+    <div class="block block__flush-top">
+        <h1>Curriculum Review</h1>
+        <p>Middle School Example Curriculum</p>
+        <p>Start the review by selecting a dimension. You do not need to complete all dimensions in one sitting. You’ll be able to download a dimension report for each dimension as well as a summary report at the end.</p>
+        <p>
+            <a href="#">
+                Content<br>
+                Covers core knowledge and skills in content standards<br>
+                6 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Utility<br>
+                Supports effective teaching<br>
+                7 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Quality<br>
+                Accurate and well presented<br>
+                5 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
+        <p>
+            <a href="#">
+                Efficacy<br>
+                Improves financial knowledge, skills, or behaviors<br>
+                3 criteria<br>
+                Est. time 30 min.
+            </a>
+        </p>
     </div>
-    <div class="block">
-        <h3>Criterion 1: Name of this criterion</h3>
-        <p>Does the curriculum address grade-level appropriate topics for saving and investing? Select yes if the following components are addressed. Select no if the following components are not addressed.</p>
-        <ol>
-            <li>
-                <h4>Indicator</h4>
+    <div class="block
+                block__flush-top
+                block__flush-bottom">
+        <h2 class="h1">{# {% include icons/document-round.svg %} #}Content</h2>
+        <div class="lead-paragraph
+                    u-mb30">
+            <p>This dimension assesses whether the curriculum content helps students develop knowledge, skills, and behaviors that are important for financial capability. Evaluation criteria are based on research and major national and state education standards.</p>
+        </div>
+        <h3 class="h2">Instructions</h3>
+        <ul>
+            <li>Read through the scope and sequence of the curriculum.</li>
+            <li>Skim the lesson plans.</li>
+            <li>Select <strong><em>yes</em></strong> for the components that are addressed, and <strong><em>no</em></strong> for those that are not.</li>
+        </ul>
+        <div class="m-notification
+                    m-notification__visible
+                    u-mb30
+                    u-mt30
+                    u-mw670">
```

Comment:
Couple things here:

1. Rather than add a new utility for this one-off thing, I think I would add the rule to a `m-notification` block in your `tdp.less` file, or possibly in a `cf-enhancements/molecules/notifications.less` file.
2. Also in that block, I think you probably want a `box-sizing: border-box;` rule. I could be making an incorrect assumption, but strongly suspect the design intent is for the rendered box width to be 670, to match the max-width on standard text elements. With standard `box-sizing`, it renders out to 702px.

Reply:
Added to cf-enhancements/molecules/notifications.less.